build.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e
 
+if [ "$(uname)" == "Darwin" ]; then
+    #Mac OS X platform
+	yes | /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
>yes | /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" [](start = 1, length = 104)

What happens with this, if Homebrew is already installed? Homebrew is pretty popular -- I expect the proportion of people on Mac that are trying to build us that also already have homebrew to be pretty high.

Now, let's consider the set of people that do *not* have it installed. The potential installation of a package management system on someone's machine as part of the simple build system, I wonder if that's perhaps going a bit too far. This is actually changing the user's system... and indeed with this `yes` command we've actually explicitly taken away that choice. This concerns me.

Reply:
I think this should be added to the machine setup doc instead. We shouldn’t be installing things during the build scripts.